unpause/unpause.go

```diff
@@ -0,0 +1,131 @@
+package unpause
```

Comment:
nit and legit question: should this be `package pause`?

Reply:
I almost named it pause but I realized that everything in it was unpause related. I suppose I can also forsee a future where there's more stuff here related to pausing (rate limits stuff); I'm happy to do whichever.

Reply:
Yeah, that was my exact thought process. I think it's fine to leave as-is for now, and rename it if/when it becomes a home for auto-pause code.